src/components/tag/filter-tag.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement, LitElement } from 'lit-element';
+import classnames from 'classnames';
+import settings from 'carbon-components/es/globals/js/settings';
+import Close16 from '@carbon/icons/lib/close/16';
+import FocusMixin from '../../globals/mixins/focus';
+import TAG_TYPE from './types';
+import styles from './tag.scss';
+
+const { prefix } = settings;
+
+/**
+ * Filter tag.
+ */
+@customElement(`${prefix}-filter-tag`)
+export default class BXFilterTag extends FocusMixin(LitElement) {
+  /**
+   * Text to show on filter tag "clear" buttons. Corresponds to the attribute with the same name
+   */
+  @property({ type: String, reflect: true })
+  title = '';
+
+  /**
+   * `true` if the tag should be disabled
+   */
+  @property({ type: Boolean, reflect: true })
+  disabled = false;
+
+  /**
+   * Specify the type of the <Tag>
+   */
+  @property({ reflect: true })
+  type = TAG_TYPE.RED;
+
+  render() {
+    const { disabled, title, type } = this;
```

Comment:
```suggestion
    const { disabled, title = 'Clear filter', type } = this;
```

Reply:
this won't be needed after [adding this change](https://github.com/carbon-design-system/carbon-custom-elements/pull/179#discussion_r349996941) right?

Reply:
That's right!